server/ws/websockets.go

```diff
@@ -24,10 +24,15 @@ type Hub interface {
 	SetReceiveWSMessage(func(data []byte))
 }
 
+type Listener struct {
```

Comment:
Everything here can be private (lowercase).

Reply:
Done.

Reply:
there is no reason to make the `Listener` struct public either.

Reply:
Done.